mesonbuild/dependencies/dev.py

```diff
@@ -497,11 +497,6 @@ def __init__(self, name: str, environment: 'Environment', kwargs: T.Dict[str, T.
 
             self.is_found = True
             self.link_args = ['-lz']
-        else:
```

Comment:
Sorry, the "else" word was not supposed to be deleted... it's needed in order to run the `libs = ...` and check for find_library and has_header stuff.